system/core/base.php

```diff
@@ -801,3 +827,34 @@ function hypha_substitute($string, array $vars) {
 		foreach ($vars as $key => $val) $string = str_replace('[[' . $key . ']]', $val, $string);
 		return $string;
 	}
+
+	/*
+		Function: hypha_getDataTypes
+		Returns an list with dataTypes
+	*/
+	function hypha_getDataTypes($excludeSystemDataTypes = false) {
```

Comment:
In a (resolved) previous comment, I said:

>AFAICS, the excludeSystemDataTypes is always passed as true. Do we really need to default to false, or even have this parameter in the first place?

Laurens said:

> It is used without a parameter in the index.php

This is true, but that actually means that the "new page" dialog (still) offers to make a `settingspage` (which is #146). I think we can just drop the exclude parameter so the function will always:
 - As a side effect, load *all* datatypes (if not done before).
 - Return only non-settings datatypes.

Reply:
done